CHANGELOG.md

```diff
@@ -11,6 +11,11 @@
 * Describe deprecated APIs in this version
 -->
 
+## [0.3.5] - 2022-04-29
+
+#### Fixed:
+* If the init method is called twice in a row, the second call should cancel the previous polling process.
```

Comment:
I wonder if instead of restarting the polling process, we should instead just `return` if `init` is called twice. It likely wouldn't happen often (if ever), but I can imagine a scenario where someone calls `init` way more than they should, and we'd be restarting the poller each time instead of just short circuit/returning. 

I don't feel real strongly about it, but I can see that being another approach here. What are your thoughts?

Reply:
That's a good alternative. Although, with that approach, if `init` was called the second time with different configuration parameters, it would ignore those config params and return the client from the first call. Not a problem now since we only have an API key parameter; maybe it would be an issue if we later added other config parameters that could change between calls.

Reply:
I prefer the option that respects the parameters of each call, even if it means each `init` call results in an RAC request.